db/schema.go

```diff
@@ -193,13 +198,13 @@ func (db *db) getCollectionsByName(
 // value.
 func substituteSchemaPatch(
 	patch jsonpatch.Patch,
-	collectionsByName map[string]client.CollectionDescription,
+	schemaByName map[string]client.SchemaDescription,
 ) (jsonpatch.Patch, error) {
-	fieldIndexesByCollection := make(map[string]map[string]int, len(collectionsByName))
-	for colName, col := range collectionsByName {
-		fieldIndexesByName := make(map[string]int, len(col.Schema.Fields))
-		fieldIndexesByCollection[colName] = fieldIndexesByName
-		for i, field := range col.Schema.Fields {
+	fieldIndexesByCollection := make(map[string]map[string]int, len(schemaByName))
```

Comment:
suggestion: `fieldIndexesBySchema`

Reply:
cheers, changing :)

- [ ] Rename var to fieldIndexesBySchema